File: data-microservice/src/main/scala/data/repository/ProjectRepository.scala
package data.repository

import data.entity.Project
import org.springframework.data.jpa.repository.{JpaRepository, JpaSpecificationExecutor}
import org.springframework.stereotype.Repository

/**
  * @author 刘佳兴
  * @version V1.0
  */
@Repository
trait ProjectRepository extends JpaRepository[Project, java.lang.Long] with JpaSpecificationExecutor[Project] {

}

File: data-microservice/src/main/java/data/repository/ProjectRepository.java
package data.repository;

import data.entity.Project;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
import org.springframework.stereotype.Repository;

/**
 * @author 刘佳兴
 * @version V1.0
 */
@Repository
public interface ProjectRepository extends JpaRepository<Project, Long>, JpaSpecificationExecutor<Project> {

}
